sibling.go

```diff
@@ -37,16 +41,23 @@ func startSibling(l log15.Logger, conn *net.UnixConn, passedFiles map[fileName]*
 		readyC: make(chan struct{}),
 		l:      l,
 	}
-	go c.writeFiles(fdNames, fds)
-	return c, nil
+	go func() {
+		err := c.writeFiles(fdNames, fds)
+		if err != nil {
+			errChan <- err
+		} else {
+			close(errChan)
```

Comment:
I know we don't need to, but it might be a good idea to close the channel in both cases? There are reasons to do it, and reasons not to do it.

Reply:
Done, it certainly looks slightly cleaner to toss in a `defer close(errChan)` imo.